core.py

```diff
@@ -2919,6 +2919,13 @@ def interpret(parent, items):
 				elif att.startswith("data-"):
 					wdg["data"][att[5:]] = val
 
+				elif att.startswith(":"):
+					attrName = att[1:]
+					attrVal = val
+
+					refObj = getattr(bindTo, attrVal)
```

Comment:
Exception raises here when both bindTo is None or attrVal is not found.

Reply:
@XeoN-GHMB please fix this to `refObj = getattr(bindTo, attrVal, None)` or add a valid error handling in case the attribute doesn't exist.